Atomic/atomic.py

```diff
@@ -915,42 +917,26 @@ def _parse_imagename(imagename):
     def _convert_to_skopeo(image):
         i = image.replace("oci:", "")
         if "http:" in i:
-            return ["--insecure", "docker://" + i.replace("http:", "")]
+            return ["--insecure"], "docker://" + i.replace("http:", "")
```

Comment:
the only code path which leads here is your system containers PR right? there's nothing in atomic 1.10 which goes here right? Asking because I haven't added that flag yet